src/ecc.rs

```diff
@@ -129,73 +151,67 @@ pub const fn make_bip32_path<const N: usize>(bytes: &[u8]) -> [u32; N] {
     let mut i = 2; // parsed character index
     let mut j = 0; // constructed path number index
     let mut acc = 0; // constructed path number
-    let mut state = BIP32ParserState::FirstDigit;
+    let mut state = Bip32ParserState::FirstDigit;
```

Comment:
Same acronym problem as for DEREncodedECDSASignature here

Reply:
OR it could just be OK because it's in the rust guidelines to have non upper case acronyms...